README.md

```diff
@@ -64,7 +85,12 @@ argument. Finally, demonstrate calling `doSomething` with a function.**
 
 Your Answer:
 ```js
-// write code here
+function doSomething(thingToDo){
+  thingsToDo="code practice";
+  console.log(thingsToDo);
+}
+// calling doSomething() will display "code practice"
+doSomething();
 ```
```

Comment:
Not quite, check out [solution](https://github.com/ga-dc/wdi7-quiz2/tree/solution)